apache/metron
# METRON-2094 Create CentOS 7 Development Environment

This creates another development environment for CentOS 7.  This work is a prerequisite for supporting HDP 3.1.  This co-exists with the other existing environments.  After some period of time, I see this replacing the CentOS 6 environment.

## Known Issues

There is currently a bug with the Ubuntu 14 environment in apache/master.   I have created [METRON-2095](https://issues.apache.org/jira/browse/METRON-2095) to track the problem.  This prevents me from fully testing the Ubuntu environment.  I have done what I can in this PR to ensure that the Ubuntu environment will continue to work once that bug is addressed. 

## Testing

1. Launch the CentOS 7 development environment.  
    * Ensure alerts are visible in the Alerts UI
    * Ensure that the Metron Service Check completes successfully.
    ```
   cd metron-deployment/development/centos7
   vagrant up
   ```

1. Launch the CentOS 6 development environment.  
    * Ensure alerts are visible in the Alerts UI
    * Ensure that the Metron Service Check completes successfully.
    ```
   cd metron-deployment/development/centos6
   vagrant up
   ```

1. Launch the Ubuntu 14 development environment.  
    * The deployment will *not* complete successfully as you will encounter [METRON-2095](https://issues.apache.org/jira/browse/METRON-2095).
    ```
   cd metron-deployment/development/ubuntu14
   vagrant up
   ```


## Pull Request Checklist
